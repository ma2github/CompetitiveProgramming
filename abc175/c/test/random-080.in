-598997128792870 114783321028966 942131582794584
