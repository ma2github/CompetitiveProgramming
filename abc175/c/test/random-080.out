598997128792870
